packages/react/src/patterns/sub-patterns/TableOfContents/__stories__/TableOfContents.stories.js

```diff
@@ -57,7 +63,22 @@ export const ManuallyDefineMenuItems = () => {
     <TableOfContents
       theme={select('theme', _themes, _themes.white)}
       menuItems={object('menuItems', menuItems)}
-      menuLabel={_menuLabel}>
+      menuLabel={_menuLabel}
+      menuRule={boolean('Optional Rule', false)}>
+      {boolean('Optional children', false) ? (
```

Comment:
@RaphaelAmadeu let's actually create this as a separate story, so it's captured in a separate percy snapshot.